augur/api/view/api.py

```diff
@@ -155,21 +155,29 @@ def user_remove_repo():
     group = request.args.get("group_name")
     repo = request.args.get("repo_id")
 
-    if not repo:
-        flash("No repo id provided")
-    if not group:
-        flash("No group name provided")
-
-    repo = int(repo)
-
-    result = current_user.remove_repo(group, repo)[0]
+    # Validate inputs early and avoid proceeding with invalid data
+    if not repo or not group:
+        if not repo:
+            flash("No repo id provided")
+        if not group:
+            flash("No group name provided")
+       
+
+    try:
+        repo_id = int(repo)
+    except (TypeError, ValueError):
+        flash("Invalid repo id provided")
```

Comment:
we should probably also print a log message here with the actual exception so the admin of augur knows what specifically happened.